src/typechecker/FStar.TypeChecker.Util.fs

```diff
@@ -2774,6 +2793,7 @@ let must_erase_for_extraction (g:env) (t:typ) =
            let bs, c = U.arrow_formals_comp t in
            let env = FStar.TypeChecker.Env.push_binders env bs in
            (U.is_ghost_effect (U.comp_effect_name c))
```

Comment:
Is this check now subsumed by the is_erasable_effect just below?

Reply:
Good catch, thanks, will change.